src/components/Form/types.ts

```diff
@@ -38,7 +39,8 @@ type ValidInputs =
     | typeof StatePicker
     | typeof RoomNameInput
     | typeof ValuePicker
-    | typeof RadioButtons;
+    | typeof RadioButtons
+    | typeof AddPlaidBankAccount;
```

Comment:
Let's adjust the comment above and remove this:
```ts
 * TODO: Add remaining inputs here once these components are migrated to Typescript:
 * EmojiPickerButtonDropdown
```

Reply:
@blazejkustra that component is not merged yet

Reply:
Lol, I thought comment referenced AddPlaidBankAccount, sorry!